definitions/divorce/json/CaseField/CaseField-nonprod.json

```diff
@@ -56,5 +64,29 @@
     "HintText": "The solicitor believes that the facts stated in this response are true",
     "FieldType": "YesOrNo",
     "SecurityClassification": "Public"
+  },
+  {
```

Comment:
I think you can remove those 3 objects: 
CoRespAgreeToCosts, CoRespStatementOfTruth and coRespondentContactDetailsConfidential as they are already in CaseField.json.
Unless there's a specific reason you needed to add them here?
At the moment as it is in both places, I think it may duplicate on the AAT's excel so it won't let you upload it to AAT :)